cluster/gce/container-linux/configure-helper.sh

```diff
@@ -814,16 +828,22 @@ function start-kube-apiserver {
     webhook_authn_config_volume="{\"name\": \"webhookauthnconfigmount\",\"hostPath\": {\"path\": \"/etc/gcp_authn.config\"}},"
   fi
 
-  params+=" --authorization-mode=RBAC"
+  local authorization_mode="RBAC"
+  if [[ -n "${ABAC_AUTHZ_FILE:-}" && -e "${ABAC_AUTHZ_FILE}" ]]; then
```

Comment:
Comment declaring intent and get @cjcullen to confirm its doing what you think and we'll still catch e2e failures.